packages/dotcom-build-code-splitting/src/bundleTypes.ts

```diff
@@ -17,6 +18,7 @@ const createSafeFileName = (moduleName: string) => {
  */
 export function createBundleWithPackages(name: string, packageNames: string[]) {
   return {
+    plugins: [new DisableTreeShakingForChunk({ test: name })],
```

Comment:
Yes, the `test` argument is used to instruct the plugin which of the generated chunks to target. It can be a chunk name, a regexp, or a function. I named the argument `test` to match Webpack but that does mean it's not completely clear what it does.

https://github.com/i-like-robots/disable-tree-shaking-for-chunk-plugin#test-string-regexp-function